_includes/articles/cardcoins-rbf-batching.md

```diff
@@ -33,11 +33,16 @@ each customer requests a withdrawal, an output is added to the transaction in
 the mempool. This transaction continues to be updated until it confirms or
 reaches some other local optimum.
 
-There are many strategies to this type of additive RBF batching. At CardCoins we
-took a safety-first approach to our implementation (with the help of Matthew
-Zipkin), the nuances of which we detailed in a more detailed blog post, RBF
-Batching at CardCoins: Diving into the Mempool's Dark Reorg Forest.
+There are many strategies to this type of additive RBF batching. At [CardCoins][] we
+took a safety-first approach to our implementation (with the help of [Matthew
+Zipkin][]), the nuances of which we detailed in a more detailed blog post, [RBF
```

Comment:
Minor edit, remove the extra 'detailed': "the nuances of which we detailed in the blog post,"